tests/bdd/features/snapshot/create/feature.feature

```diff
@@ -0,0 +1,78 @@
+Feature: Volume Snapshot Creation
+
+  Background:
+    Given a deployer cluster
+
+  Scenario Outline: Snapshot creation of a single replica volume
+    Given we have a single replica <publish_status> volume where the replica is <replica_location> to the target
+    When we try to create a snapshot for the volume
+    Then the snapshot creation should be successful
+    And the snapshot status ready to clone should be true
+    And the snapshot source size should be equal to the volume size
+    And the snapshot size should be equal to the volume size
+    And the snapshot timestamp should be in the RFC 3339 format
+    And the snapshot timestamp should be within 1 minute of creation (UTC)
```

Comment:
We don't really need to mention timezone UTC here specifically, as we are talking about a delta.

Reply:
What do we consider "creation" time here? The timestamp at which test will call put_volume_snapshot?

Reply:
> We don't really need to mention timezone UTC here specifically, as we are talking about a delta.
We might do because if your computer is not on UTC then you can't just compare time as the snapshot time is in utc?

> What do we consider "creation" time here? The timestamp at which test will call put_volume_snapshot?
It doesn't matter too much if it's right before we issue the call or after, 1 minute will be a large enough slack for work for both cases